### ISSUE TITLE
English version

### ISSUE THREAD

**Issue by pold500** [Nekrolm/ubbook#101]
Было бы неплохо завести английскую версию, все-таки, по понятным причинам. Шерить на всяких западных сайтах и реддитах.
Могу помочь с переводом.

**Comment by Nekrolm** [comment]
Да, неплохо было бы. Это довольно большая работа, на которую у меня не всегда есть время
